R/rwebppl.R

```diff
@@ -343,21 +344,20 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
   if (file.exists(error_file)) {
     file.remove(error_file)
   }
+  finish_file <- sprintf("/tmp/webppl_finished_%s", uid)
+  finish_arg <- finish_file
```

Comment:
My understanding is that `error_file` is the only one that doesn't get a uid and therefore the only one that is cleaned and rewritten every time. I guess there are two solutions: (1) only clean the error file, or (2) switch it to using a uid so that we don't need to do cleaning at all. Seems like the second option is better?